calcom/cal.com
# chore: App-router-migration`(/auth/verify`) add missing metadata / page & extract components into `/module`

## What does this PR do?



- Fixes #16295
- Fixes CAL-4218
- Remove references to `/pages` for `/auth/verify`, `/auth/verify-email` and `/auth/verify-email-change`
- Add missing metadata to these 3 pages in app router



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Navigate to:

1. `future/auth/verify`
2. `future/auth/verify-email-change`
4. `/auth/verify`
5. `/auth/verify-email-change`
